tests/testing_four_c_simulation.py

```diff
@@ -109,6 +115,29 @@ def run_four_c_test(self, name, mesh, n_proc=2, restart=[None, None], **kwargs):
         )
         self.assertEqual(0, return_code, msg="Test {} failed!".format(name))
 
+    def create_cantilver_model(self, n_steps, time_step=0.5):
```

Comment:
Can we move this out of the unit test class as a free function?

Reply:
Yes, I would actually like to move it into a complete new file named something as `mesh_generations_for_testing.py`. I think it would be good to store some "default meshes" for testing in one file.